### ISSUE TITLE
Create effective SEO for site to make it searchable online.

### ISSUE THREAD

**Issue by nikolajbaer** [opensandiego/sandiego-food-map#19]
React sites can be tricky to get indexed, but we want to make sure we have information about our site in the language that people might be searching for so potential needy users can find the site easily on a simple google search.

**Comment by nikolajbaer** [comment]
- [x] Identify the right schema.org microdata to embed on the site

**Comment by BolunThompson** [comment]
schema.org has a [map](https://schema.org/Map) microdata format, which should be the correct one to use.

**Comment by BolunThompson** [comment]
I don't believe indexing should be an issue, as [google should index](https://developers.google.com/search/docs/guides/javascript-seo-basics) them, albeit with a delay.

As for finding this on search, to an extent I don't feel like that is a problem if we're mentioned on San Diego Food Bank's website;  searching for food resources should often lead to their website. As for search terms that don't lead to their website, ["free food san diego"](https://trends.google.com/trends/explore?geo=US&q=san%20diego%20food%20map,free%20food%20san%20diego,san%20diego%20food%20pantry,free%20meals%20san%20diego,san%20diego%20food%20bank) seems to be a semipopular search term, so something along those lines may be worth mentioning on the website (for what it's worth, [Food Oasis](https://foodoasis.la/) uses similar language). 

I'm don't have much experience with SEO however, so this may be worth revisiting in the future.
